TF Security (#2087)
saucelabs/sauce-docs: docs/test-fairy/security/hiding-data.md

<TabItem value="android">

To hide a view from video, all you need to do is the following:

```java
TestFairy.hideView(Integer.valueOf(R.id.my_view));
```

or

```java
TestFairy.hideView(View myView);
```

Replace `R.id.my_view` with the identifier of the view you wish to hide. Review the full example below:

Example

```java
public class MyActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        setContentView(R.layout.my_activity);

        TestFairy.hideView(findViewById(R.id.credit_card));
    }
}
```

</TabItem>

<TabItem value="ios">

To hide a view from video, all you need to do is call the static instance method hideView in the TestFairy class:

```js
UIView *view = ...
[TestFairy hideView:view];
```

Example

```js
@interface MyViewController : UIViewController {
    IBOutlet UITextField *usernameView;
    IBOutlet UITextField *creditCardView;
    IBOutlet UITextField *cvvView;
}

@implementation MyViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    [TestFairy hideView:creditCardView];
    [TestFairy hideView:cvvView];
}
```

</TabItem>

<TabItem value="react">

> In order to hide views from your recorded session, you will need to pass a reference to a view to TestFairy. First, give the element to be hidden a ref attribute. For example:

Author: ```suggestion
To hide views from your recorded session, you must pass a reference to a view to TestFairy. First, give the element to be hidden as a ref attribute. For example:
```
